content/en/03-using-pixie/02-using-cli.md

```diff
@@ -142,7 +144,23 @@ px run -f <script.pxl>		# Use Pixie's CLI to run the pxl script with the provide
 px live -f <script.pxl>		# Use Pixie's Live CLI to run the pxl script with the provided filename
 ```
 
-## Debugging Pixie using the CLI
+## Advanced
+
+### Disabling Encryption
+
+By default, Pixie implements end-to-end encryption for telemetry data requested by the CLI. For more information, see the [FAQ](/about-pixie/faq#how-does-pixie-secure-its-data).
```

Comment:
Do we call it `telemetry data` elsewhere? I called it `script data` in our API release.
Also drop the `By default` here since the defaults might change and we probably want to publish this independently of when we change the default behavior.

Reply:
We currently use telemetry data throughout the docs (no instances of "script data"), so I'll keep it consistent unless you strongly prefer else

Reply:
Consistency sounds good